dapps/tests/app/test/token_spec.js

```diff
@@ -42,7 +42,9 @@ config({
         }
       },
       SomeContract: {
-        deployIf: "await MyToken.methods.isAvailable().call()",
+        deployIf: (dependencies) => {
+          return dependencies.contract.MyToken.methods.isAvailable().call();
+        },
```

Comment:
Did we turn off string support in deployment hooks? I'm generally all for it, but I don't see a reference as to why this change here is needed otherwise.

Reply:
No, it's just that we thought we had a race condition between the execution of the `deployIf`s/`onDeploy`s and the execution of the tests, so changing the hooks to functions should have fixed that. 
Sadly, it didn't have the wanted effect, but at least, the function syntax is nicer